staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go

```diff
@@ -103,10 +103,11 @@ func (m *PathRecorderMux) ListedPaths(clusterName string) []string {
 }
 
 func (m *PathRecorderMux) trackCallers(path string) {
+	stack := string(debug.Stack())
 	if existingStack, ok := m.pathStacks[path]; ok {
-		utilruntime.HandleError(fmt.Errorf("registered %q from %v", path, existingStack))
+		utilruntime.HandleError(fmt.Errorf("duplicate path registration of %q: original registration from %v\n\nnew registration from %v", path, existingStack, stack))
```

Comment:
I should probably submit this separately to k/k - it's quite difficult to determine duplicate registrations if the error message only tells you where the new (duplicated) registration comes from.

Reply:
+1 for k/k

Reply:
https://github.com/kubernetes/kubernetes/pull/106892

Reply:
tagged